test/api/graph.js

```diff
@@ -1,3 +1,19 @@
+Skip to content
```

Comment:
Hmm, I think something isn't right here.

Also, we probably don't need to have the graph tests for `createEmbed`, we are reliably testing that method in another place.